websites/views_test.py

```diff
@@ -702,6 +707,7 @@ def test_websites_content_list(  # pylint: disable=too-many-locals
             website=website,
             title=f"some TEXT{num} here for a case insensitive search",
             metadata={"resourcetype": "Image" if num % 2 == 0 else "Video"},
+            file=f"myFile{num+1}.png",
```

Comment:
Include a path here to make sure it is being filtered out from search results:
```
file=f"courses/my-test-site/myFile{num+1}.png",
```

Reply:
Updated in 929092e405c3630d744d9e3e0b96e1e9129fb513.